Dockerfile

```
@@ -1,46 +1,54 @@
 ENV UV_PYTHON=python3.12
 ENV UV_COMPILE_BYTECODE=1
 ENV UV_NO_SYNC=1
 ENV UV_CACHE_DIR=${APP_DIR}/.cache
+ENV UV_NO_CACHE=1
+# Prevents Python from writing .pyc files to disk
+ENV PYTHONDONTWRITEBYTECODE=1
+# Ensures Python output is logged straight to the terminal (useful for OpenShift logs)
+ENV PYTHONUNBUFFERED=1
 
-RUN dnf -y install \
+RUN microdnf -y install \
```

Hadolint (2.14.0)

[warning] Specify version with `dnf install -y <package>-<version>`.

(DL3041)